test/controllers/patients_controller_test.rb

```diff
@@ -33,8 +33,9 @@ class PatientsControllerTest < ActionDispatch::IntegrationTest
 
     it 'should not serve html' do
       sign_in @data_volunteer
-      get patients_path
-      assert_response :not_acceptable
+      assert_raise ActionController::UnknownFormat do
```

Comment:
hmmm, this assertion should probably stay. I think if you take the test and change it to `get patients_path, xhr: true` that'll make it blend, that seemed to work in a bunch of the other controller tests. (Relatedly, I think this change is my fault, haha.)